src/app/Mail/SimpleMail.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+namespace App\Mail;
+
+use Illuminate\Bus\Queueable;
+use Illuminate\Mail\Mailable;
+use Illuminate\Queue\SerializesModels;
+
+class SimpleMail extends Mailable
+{
+    use Queueable, SerializesModels;
+
+    public $messageText;
+    public $recipient;
+    public $viewTemplate;
+
+    public function __construct($messageText, $recipient, $viewTemplate)
```

Comment:
Otypoval by som parametre a vygeneroval PHPDoc (to vie spravit aj Intellij, potom tam len staci dopisat, tromi slovami, co robi tato metoda)

Reply:
Spravené.